app/controllers/admin/financials/market_payments_controller.rb

```diff
@@ -3,6 +3,7 @@ class MarketPaymentsController < AdminController
     before_action :require_admin
 
     def index
+      # dead_code!
```

Comment:
Is this meant to be commented out?

Reply:
Only real comment is this one.

Reply:
I was just using this as an example of usage. I can update